migrations/20220809015149-create-program-location.js

```diff
@@ -8,10 +8,10 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.INTEGER,
       },
-      LocationUserId: {
+      LocationId: {
         type: Sequelize.INTEGER,
       },
-      VisitorsUserId: {
+      VisitorsId: {
```

Comment:
Again, I'm not sure what `VisitorsId` is referring to... is this not the join model between `Program` and `Location` for that many-to-many relationship? Shouldn't there be a `ProgramId` since the other one is `LocationId`?